src/services/exprensesService.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+const { getAllUsers } = require('../services/usersServer');
+
+let expenses = [];
+
+getAllUsers().forEach((el, index) => {
+  const date = new Date();
+
+  expenses.push({
+    id: uuidv4(),
+    userId: el.id,
+    spentAt: new Date(date.getTime() + index * 10000),
+    title: 'Test',
+    amount: 10000,
+    category: 'salary',
+  });
+});
```

Comment:
remove it to pass the test
![image](https://user-images.githubusercontent.com/98764641/236407639-5dc4d4fd-55fa-4942-a166-81b5b9df33e1.png)

Reply:
done